### ISSUE TITLE
Accessibility - Non text contrast (AA)

### ISSUE THREAD

**Issue by ainemckay** [MHRA/products#921]
Active user interface components and meaningful non text graphics should be distinguishable by people with visual impairments. The minimum colour contrast for
such components is 3:1.
Issue
Some form borders did not meet the minimum colour contrast ratio of 3:1.
1. Grey (#9A9B9C) form border on light blue (#E6EAF4) (All MHRA Products pages)

Recommendation
Colour contrast for interactive elements should meet the minimum colour contrast
ratio 3:1.
 Change border to black

**Comment by TimboTambo** [comment]
Release:
- Deploy latest changes to products site using automated release workflow
